packages/orchestration/src/exos/local-chain-facade.js

```diff
@@ -38,17 +41,25 @@ const prepareLocalChainFacadeKit = (
   zone,
   {
     makeLocalOrchestrationAccountKit,
+    agoricNames,
     localchain,
     // TODO vstorage design https://github.com/Agoric/agoric-sdk/issues/9066
     // consider making an `accounts` childNode
     storageNode,
-    vowTools: { allVows, watch },
+    vowTools: { allVows, watch, asVow },
   },
 ) =>
   zone.exoClassKit(
     'LocalChainFacade',
     {
-      public: ChainFacadeI,
+      public: M.interface('LocalChainFacade', {
+        getChainInfo: M.call().returns(VowShape),
+        makeAccount: M.call().returns(VowShape),
```

Comment:
please make apparent the composition
```suggestion
      public: M.interface('LocalChainFacade', {
        ...ChainFacadeI,
```

Reply:
hm... that doesn't quite work; `ChainFacadeI` is an `M.interface()`, not a record. I'll check to see if there's a way to get the methods record out of an interface.

Reply:
oh right right. here's what we did last time https://github.com/Agoric/agoric-sdk/blob/a724e4ace9e70a58a3546a9f80f41f9489da496e/packages/orchestration/src/exos/cosmos-orchestration-account.js#L70-L73